tests/test_datetimes.py

```diff
@@ -456,9 +454,8 @@ def test_nested_context_manager():
 def _assert_datetime_date_and_time_are_all_equal(expected_datetime):
     assert datetime.datetime.now() == expected_datetime
     assert datetime.date.today() == expected_datetime.date()
-    datetime_from_time = datetime.datetime.fromtimestamp(time.time())
-    timezone_adjusted_datetime = datetime_from_time + datetime.timedelta(seconds=time.timezone)
-    assert timezone_adjusted_datetime == expected_datetime
+    datetime_from_time = datetime.datetime.utcfromtimestamp(time.time())
```

Comment:
I'm not convinced that this is the right thing to do, but I also don't think the assertion this replaces makes much sense.

It seems to be asserting that `datetime.datetime.now()` should *not* be equal to `datetime.fromtimestamp(time.time())` if the test is run in a locale with an offset, which is not the same behavior as what `datetime.datetime` does. Fixing this assumption in the tests (and code?) is how this should be fixed, I think.